src/io/flutter/sdk/FlutterSdkUtil.java

```diff
@@ -49,9 +49,9 @@ private FlutterSdkUtil() {
   /**
    * Return the environment variable value to use when shelling out to the Flutter command-line tool.
    */
-  public static String getFlutterEnvValue() {
+  public static String getFlutterHostEnvValue() {
     final String clientId = ApplicationNamesInfo.getInstance().getFullProductName().replaceAll(" ", "-");
```

Comment:
This is just normalizing what we're sending to the flutter cli - we'll mostly have control over what is send into the tool (we'll send it in from IntelliJ; similar modifications to the travis testing script and other callers will need to be made).